### ISSUE TITLE
[Feature request] Disable 15 second warning message

### ISSUE THREAD

**Issue by vatterspun** [bingmann/cryptote#2]
I know this program probably isn't in development anymore, but I thought I'd throw out a few things I'd been hoping to see.  The big one is the warning msg.

![image](https://user-images.githubusercontent.com/1289134/27893671-c02d31ec-61cc-11e7-98f2-a83066459d2f.png)

It makes sense to maybe have this appear on the "first run" of the program but not afterwards.

**Comment by ghost** [comment]
This would be an awesome feature as the wait every time is currently hurting the usability for no reason. Would you like me to implement the change?

**Comment by bingmann** [comment]
??? This is only shown between the two password entries, when adding a password.
```
        // Show legal notice dialog in between entries
        WLegalNotice dlglegal(this);
        dlglegal.ShowModal();
```

**Comment by ghost** [comment]
True, but when you deal with multiple files in different locations that will all be in individual containers, it is n * 15 seconds which can  be quite a lot of time. I just encrypted 30+ files individually.

I am not sure about the legal validity of the dialog (IANAL), but from usability point of view it is really annoying and slows down dramatically using your otherwise excellent application.

My proposal is to show the dialog only once when user starts the application for the first time, or when a container is saved for the first time, and afterwards just leave it out.

Cheers!

**Comment by vatterspun** [comment]
> Would you like me to implement the change?

If you're willing to take over development then yes.  However, if you just want to make one update, I'd skip it.  Thanks for asking.

**Comment by ghost** [comment]
Hopefully Timo will implement it.
